core/src/main/java/com/avast/bytes/BytesComparator.java

```diff
@@ -0,0 +1,17 @@
+package com.avast.bytes;
+
+import java.util.Comparator;
+
+/**
+ * Natural Bytes comparator that orders Bytes as array of signed bytes.
```

Comment:
Please, remove the word "natural". Both me @jendakol think that it's unnecessary and a bit misleading. Thanks.